docs/contributing.md

```diff
@@ -35,7 +35,7 @@ To submit a pull request, please use the following workflow:
 3. keep your feature branch rebased and up-to-date with the `scores` develop branch,
 4. when ready, submit a pull request to the develop branch of `scores`.
 
-**Note for new contributors:** we are still establishing the smoothest path for new contributors to make code contributions. If you aren't sure where or how to start, please email scores@bom.gov.au and we would be happy to help discuss your goals and work through any issues getting set up.
+Some developers like to prefix their branch names with a few digits to help disambiguate them. This is up to the developer and any approach to naming is welcome. If you have any questions about the development process, please feel free to email scores@bom.gov.au and someone from the development team will respond to your query.
```

Comment:
I think it would be better to direct people to https://github.com/nci/scores/discussions to ask their questions rather than email. It would be good for these questions to be answered in the open in case others have those questions (and we keep the email more for reporting security issues)

Reply:
Thanks. I've pushed up a revised version, let me know what you think. (Steph helped).

Reply:
Looks good

Reply:
A quick note, some of the changes didn't get included in Tennessee's earlier commit. He has just pushed another commit.